launcher/commands/install.py

```diff
@@ -173,6 +173,18 @@ def _create_full_install_args() -> Dict:
 
     return arguments
 
+def replace_string_in_file(file_path, target_string, replacement_string):
+    # Read the contents of the file
+    with open(file_path, 'r') as file:
+        file_contents = file.read()
+
+    # Replace the target string with the replacement string
+    modified_contents = file_contents.replace(target_string, replacement_string)
+
+    # Write the modified content back to the file
+    with open(file_path, 'w') as file:
```

Comment:
file_path is a Path so you can use `file_path.write_text()`

In the end, this function is resumed at:
```python
file_path.write_text(
     file_path.read_text().replace(target_string, replacement_string)
)
```

Reply:
I will keep it on multiple lines to keep it more readable.